### PR TITLE
Simplify Razor bug reports.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
- We now have better mechanisms for capturing information so no need to scare users off.

### CODE REVIEW

**Review by Pilchie — CHANGES_REQUESTED** [review]
What's with the submodule update?

File: .github/ISSUE_TEMPLATE/30_razor_tooling.md [hunk 1]
```diff
@@ -15,38 +15,16 @@ For other types of questions, consider using [StackOverflow](https://stackoverfl
 
 -->
 
-<!-- NOTE: This issue template is meant specifically to be used for issues with the new experimental Razor tooling experience provided in Visual Studio's Preview Feature pane -->
+### Visual Studio Code
 
-### Describe the bug
-A clear and concise description of what the bug is.
+Select the `Razor: Report a Razor Issue` command in VSCode (`ctrl + shift + p`) and follow the instructions to gather information necessary for us to investigate. Once done you can replace this content with contents that are put on your clipboard.
```
Comment (TanayParikh) [inline .github/ISSUE_TEMPLATE/30_razor_tooling.md:20]:
```suggestion
Select the `Razor: Report a Razor Issue` command in VSCode (<kbd>CTRL</kbd> + <kbd>SHIFT</kbd> + <kbd>P</kbd>) and follow the instructions to gather information necessary for us to investigate. Paste below:

---
<Paste Content Here>
---

```

File: .github/ISSUE_TEMPLATE/30_razor_tooling.md [hunk 2]
```diff
@@ -15,38 +15,16 @@ For other types of questions, consider using [StackOverflow](https://stackoverfl
...
 
-#### Activity log (only needed if VS crashes)
-[Here](https://docs.microsoft.com/en-us/visualstudio/extensibility/how-to-use-the-activity-log?view=vs-2019#to-examine-the-activity-log) are the instructions on how to generate/acquire one. Note that GitHub does not generally allow .xml files to be uploaded with issues.
+You may file an issue here but please create a Visual Studio bug first and link here as it will capture all necessary logs for us to investigate further.
 
-#### Language Server logs
-1. Run Visual Studio with the [/Log](https://docs.microsoft.com/en-us/visualstudio/ide/reference/log-devenv-exe?view=vs-2019) command line switch
-2. Reproduce the issue
-3. Provide the logs located at `%Temp%\VisualStudio\LSP`
+How to report a problem in Visual Studio: https://docs.microsoft.com/en-us/visualstudio/ide/how-to-report-a-problem-with-visual-studio?view=vs-2019
```
Comment (TanayParikh) [inline .github/ISSUE_TEMPLATE/30_razor_tooling.md:26]:
```suggestion
How to report a problem in Visual Studio: https://docs.microsoft.com/en-us/visualstudio/ide/how-to-report-a-problem-with-visual-studio
```

so we don't have to maintain the link 😄

**Review by TanayParikh — COMMENTED** [review]
LGTM aside from the submodules Kevin mentioned 😄

**Review by ryanbrandenburg — APPROVED** [review]

**Comment by NTaylorMullen** [comment]
> What's with the submodule update?

lol, total mishap 😄

**Review by Pilchie — APPROVED** [review]

**Review by TanayParikh — APPROVED** [review]

**Pull request merged**